diff -ruN vlc/modules/hw/nvdec/meson.build vlc-patched/modules/hw/nvdec/meson.build
--- vlc/modules/hw/nvdec/meson.build	2024-12-07 21:29:20.863891799 -0300
+++ vlc-patched/modules/hw/nvdec/meson.build	2024-12-07 21:36:41.873859068 -0300
@@ -1,9 +1,9 @@
 if get_option('nvdec').allowed()
 nvdec_dep = disabler()
 if cc.check_header('ffnvcodec/dynlink_loader.h')
-    nvdec_dep = enabler()
+#    nvdec_dep = enabler()
 elif cc.check_header('ffnvcodec/dynlink_loader.h', dependencies: contrib_dep )
-    nvdec_dep = contrib_dep
+#    nvdec_dep = contrib_dep
 endif
 endif
 vlc_modules += {
